_api-reference/ingest-apis/ingest-pipelines.md

```diff
@@ -0,0 +1,390 @@
+---
+layout: default
+title: Ingest pipelines
+parent: Ingest APIs
+nav_order: 10
+redirect_from:
+  - /opensearch/rest-api/ingest-apis/create-update-ingest/
+  - /api-reference/ingest-apis/create-update-ingest/
+  - /api-reference/ingest-apis/get-ingest/
+  - /api-reference/ingest-apis/simulate-ingest/
+  - /api-reference/ingest-apis/delete-ingest/
+---
+
+# Ingest pipelines
+
+Ingest pipelines in OpenSearch are managed by using ingest API operations. When using ingest in production environments, your cluster should contain at least one node with the node roles permission set to `ingest`. For more information about setting up node roles within a cluster, see [Cluster formation]({{site.url}}{{site.baseurl}}/opensearch/cluster/).
```

Comment:
`ingest` is a role of a node but not permission.

```suggestion
Ingest pipelines in OpenSearch are managed by using ingest API operations. When using ingest in production environments, your cluster should contain at least one node with the `ingest` role. For more information about setting up node roles within a cluster, see [Cluster formation]({{site.url}}{{site.baseurl}}/opensearch/cluster/).
```

Reply:
Changed in PR #4299